tasks/image-segmentation/task.yaml

```diff
@@ -1,16 +1,21 @@
 id: image-segmentation
-name: Image Segmentation
+name: image-segmentation
```

Comment:
The `name` field can be completely removed. `id` gets used as the name now, after more recent changes.